lib/webenginesettings.cpp

```diff
@@ -60,8 +61,12 @@ void SailfishOS::WebEngineSettings::initialize()
     engineSettings->setPreference(QStringLiteral("embedlite.azpc.handle.longtap"), QVariant::fromValue<bool>(false));
     engineSettings->setPreference(QStringLiteral("embedlite.azpc.json.longtap"), QVariant::fromValue<bool>(true));
     engineSettings->setPreference(QStringLiteral("embedlite.azpc.json.viewport"), QVariant::fromValue<bool>(true));
+    // TODO: Fix this so that it can be applied during runtime when QQuickItem based WebView is used with QQuickFlickable.
+    // At the moment just disable it. JB#39581
```

Comment:
maybe "At the moment just disable it, to avoid unnecessary events being fired.  JB#39581"

Reply:
Fixed